examples/structured_2d_dgsem/elixir_eulerisothermal_wave.jl

```diff
@@ -0,0 +1,83 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the polytropic Euler equations
+
+gamma = 1.0
+kappa = 1.0     # Scaling factor for the pressure.
```

Comment:
In all the new elixirs, you never set `kappa` to a value other than one. For the convergence test maybe set `kappa` to be something else, it is a toy problem anyway.

Reply:
OK. Change kappa for this test.